<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>20</y>
      <width>200</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>191</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Add element</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>470</y>
       <width>191</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Sort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Visualisation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>191</width>
       <height>131</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>bubble</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>shaker</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>heap</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>quick</string>
      </property>
     </item>
    </widget>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>191</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>191</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Random</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>701</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>20</y>
      <width>191</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="listWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>470</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>bubble</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>shaker</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>heap</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>quick</string>
      </property>
     </item>
    </widget>
    <widget class="QListWidget" name="listWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>bubble</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>shaker</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>heap</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>quick</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
